content/realtime/channels-messages.textile

```diff
@@ -421,9 +433,30 @@ bc[swift]. realtime.channels.get("private:chatroom").attach { error in
     }
 }
 
+
+h4(#fatal-errors). Fatal channel errors
+
+Some classes of errors are fatal. These cause the channel to move to the @FAILED@ state. The client library will not attempt any automatic recovery actions. For example: attempting to attach to a channel that the token you're using does not have the @subscribe@ capability for will cause that channel to enter the @FAILED@ state.
+
+Note that while fatal errors won't get better on their own, they can be fixable. For example, if a channel goes into the @FAILED@ state due to the client not having the right capabilities to attach to it, that client could use "@Auth#authorize()@":/realtime/authentication#authorize to obtain a new token which does have the right capabilities, then call "@attach()@":/channels-messages#attach on the channel. While the library will not automatically reattach in the @FAILED@ state, explicit calls to "@attach()@":/realtime/channels-messages#attach will make the client try again.
+
+h4(#nonfatal-errors). Nonfatal errors
+
+Other types of error are nonfatal. For example, a client may have network connectivity issues, or a channel may experince a loss of strict message continuity. The library will automatically recover from these events. If channel continuity is lost in the process, the library will notify you though a @resumed@ flag in the @ATTACHED@ event, so you can decide how to handle that failure.
```

Comment:
s/experince/experience/

and s/The library will automatically recover/The library will automatically attempt to recover/